.stylelintrc

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "stylelint-config-standard-scss"
+    ],
+    "plugins": [
+        "stylelint-selector-bem-pattern"
+    ],
+    "ignoreFiles": ["sass/tna-toolkit/**/*.scss"],
+    "rules": {
+        "at-rule-empty-line-before": null,
+        "block-no-empty": null,
+        "declaration-empty-line-before": null,
+        "property-no-vendor-prefix": null,
```

Comment:
Just out of interest, which properties do we have that are vendor prefixed?

Reply:
I saw a few being removed when the linting rules didn't have that - here are a few examples

https://github.com/nationalarchives/ds-wagtail/blob/cfa2dd90a25764fd4d03f8749a87be8dd610bf3b/sass/tna-toolkit/bootstrap/_reboot.scss#L374-L379

https://github.com/nationalarchives/ds-wagtail/blob/cfa2dd90a25764fd4d03f8749a87be8dd610bf3b/sass/tna-toolkit/bootstrap/_tables.scss#L170-L172

https://github.com/nationalarchives/ds-wagtail/blob/cfa2dd90a25764fd4d03f8749a87be8dd610bf3b/sass/includes/search/_featured-search.scss#L10-L29

Reply:
Okay - those are some interesting examples! 😅

The first one isn't even a supported property any more and the second was to fix an issue from 10 years ago. Both are just Bootstrap and that will be going at some point.

The third one is difficult because MSIE's grid rendering was fundamentally different from everyone else's and couldn't just be solved with an `-ms-` prefix. We'll have to look at improving that one at some point.

Thank you for the examples. We'll leave the prefixed versions in and remove the `property-no-vendor-prefix` rule for now to make things easier. 👍🏼

Reply:
autoprefixer might be a good solution for ms-grid.

Reply:
It might be if we do look to keep the grid stuff from Bootstrap. I wouldn't worry about it too much now though.